core/input/input.cpp

```diff
@@ -731,6 +748,202 @@ Vector3 Input::get_gyroscope() const {
 	return gyroscope;
 }
 
+bool Input::is_joy_accelerometer_enabled(int p_device) const {
+	_THREAD_SAFE_METHOD_
+	return joy_motion.has(p_device) && joy_motion[p_device].accelerometer_enabled;
+}
+
+bool Input::is_joy_gyroscope_enabled(int p_device) const {
+	_THREAD_SAFE_METHOD_
+	return joy_motion.has(p_device) && joy_motion[p_device].gyroscope_enabled;
+}
+
+Vector3 Input::get_joy_accelerometer(int p_device) const {
+	_THREAD_SAFE_METHOD_
+	if (!joy_motion.has(p_device)) {
+		return Vector3();
+	}
+
+	if (!joy_motion[p_device].calibration.calibrated) {
+		return joy_motion[p_device].accelerometer;
+	}
+
+	const MotionInfo &motion = joy_motion[p_device];
+	// Calibrated accelerometer doesn't include gravity
+	Vector3 value = (motion.accelerometer - motion.gravity) - motion.calibration.accelerometer_offset;
+	if (std::abs(value.x) < motion.calibration.accelerometer_deadzone) {
```

Comment:
```suggestion
	if (Math::abs(value.x) < motion.calibration.accelerometer_deadzone) {
```
For all these, please use the built-in methods